LayoutTests/imported/w3c/web-platform-tests/css/css-overscroll-behavior/overscroll-behavior-expected.txt

```diff
@@ -9,7 +8,5 @@ Repeat the same scrolls as in step 1 and then tap on DONE.
 Repeat the same scrolls as in step 1 and then tap on DONE.
 Make two separate scrolls on GREEN, in this order: scroll UP (or drag down), then scroll LEFT (or drag right). Scroll (or drag) until nothing is scrolling. Then tap on DONE.
 
-Harness Error (FAIL), message = Unhandled rejection: Can't find variable: action
-
-NOTRUN overscroll-behavior prevents scroll-propagation in the area and direction as specified
+PASS overscroll-behavior prevents scroll-propagation in the area and direction as specified
```

Comment:
This doesn't match the behavior upstream that WebDriver implements: https://wpt.fyi/results/css/css-overscroll-behavior?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-scrolling%20css%2Fcss-overscroll-behavior%2Foverscroll-behavior

I think consistency is important here.

Reply:
Yeah I have tried investigating this a bit with a local WebDriver and the behavior on this test is strange. If I remove the overscroll-behavior values set it is expected that both scrolls would propagate, but that is not the behavior I was seeing (only the vertical scroll was propagating). I also noticed that for vertical wheel events we were getting multiple opposite events (positive then negative events). I'm still trying to understand where along the flow things are broken. I'm hoping to come back to this soon.